beamer/tests/agent/test_handlers.py

```diff
@@ -228,3 +229,26 @@ def test_handle_request_resolved(token: Contract, config: Config):
     assert request.l1_resolution_filler is None
     assert process_event(event, context) == (True, None)
     assert request.l1_resolution_filler == filler
+
+
+def test_handle_claim_made(token: Contract, config: Config):
+    context = make_context(config)
+
+    request = make_request(token)
+    request.fill(config.account.address, b"", b"")
+    context.requests.add(request.id, request)
+
+    claim = make_claim(request, claimer=config.account.address)
+    context.claims.add(claim.id, claim)
+
+    event = deepcopy(claim._latest_claim_made)
```

Comment:
:warning: private attribute access. Please use public API (if there is not one, consider adding it).